README.md

```diff
@@ -1,12 +1,66 @@
-TutorialWorldGeneration
-=======================
+<div align="center">
+<img src="./docs/_media/banner.png">
+</div>
 
-A tutorial on the facet world generation system
+# Tutorial: World Generation
 
-See the wiki for a [tutorial](https://github.com/Terasology/TutorialWorldGeneration/wiki)
+_This is a tutorial module for [Terasology].
+It explains the concepts of world generation and how to apply them to implement new world generators._
 
-![Facet Production](/images/Facet%20Production.png)
-![Noise Sampling](/images/Noise%20Sampling.png)
-![Facet Modification](/images/Facet%20Modification.png)
-![Houses](/images/Houses.png)
-![Lakes](/images/PluginLakes.jpg)
+<h2 align="center"><a href="https://terasology.github.io/Health">👉 Tutorial 👈</a></h2>
```

Comment:
This links to the Health module, not TutorialWorldGeneration; it also should maybe link to the Tutorial subpage of the site, instead of the homepage which is just this README.